tools/hosting/docker-compose.yml

```diff
@@ -58,7 +58,7 @@ services:
 
   # FossilDB
   fossildb:
-    image: scalableminds/fossildb:master__504
+    image: scalableminds/fossildb:master__510
```

Comment:
Do you also need to change the column families here?

Reply:
No, since this yaml still points to the old webknossos release 24.12.0.

We’ll update it eventually to a newer release, then we will have to check the migration guide including the changed columnFamilies.